Zend/tests/bug40784.phpt

```diff
@@ -1,24 +0,0 @@
---TEST--
-Bug #40784 (Case sensivity in constructor's fallback)
---FILE--
-<?php
-
-class A {
-	  function A () { echo "I'm A\n"; }
-}
-
-class B extends A {
-  function __construct() {
-    parent::__construct();
-    parent::__constrUct();
-  }
-}
-
-$b = new B;
-
-echo "Done\n";
-?>
---EXPECTF--	
-I'm A
-I'm A
-Done
```

Comment:
I think these are left-over from the earlier RFC with removed them outright and I didn't notice it. You are correct; we should probably keep these until old-style constructors are removed.